internal/pkg/archer/app.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package archer
+
+// Application represents a deployable service or task.
+type Application struct {
+	Project string `json:"project"` // Name of the project this application belongs to.
+	Name    string `json:"name"`    // Name of the application, which must be unique within a project.
+	Type    string `json:"type"`    // Type of the application (LoadBalanced app, etc)
+}
+
+// ApplicationStore can List, Create and Get applications in an underlying project management store
+type ApplicationStore interface {
```

Comment:
I suppose we don't have delete just to be safe? Will customers want to update their app? Like rename it or something.

Reply:
We'll add Delete in mile stone 3! #